src/main/java/com/sap/broker/budgie/domain/Binding.java

```diff
@@ -0,0 +1,51 @@
+package com.sap.broker.budgie.domain;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Map;
+import java.util.UUID;
+
+public class Binding {
```

Comment:
Is this really binding? Is that service instance or just service?
Binding is something that represents the connection between given application and a service.

Reply:
The purpose of this class is to hold the meta information about the binding. The bindings this service broker provides are dummy ones.

Reply:
Yes but again, the Binding you represent here is not a real one - even though you are using this class as a Mocked class.